op-wheel/commands.go

```diff
@@ -61,6 +66,23 @@ var (
 		TakesFile: true,
 		EnvVars:   prefixEnvVars("ENGINE_JWT_SECRET"),
 	}
+	EngineOpenEndpoint = &cli.StringFlag{
+		Name:    "engine.open",
+		Usage:   "Open Engine API RPC endpoint, can be HTTP/WS/IPC",
+		Value:   "http://localhost:8545/",
+		EnvVars: prefixEnvVars("ENGINE_OPEN"),
+	}
+	EngineVersion = &cli.IntFlag{
```

Comment:
Do we want to set a default version here?

Reply:
I wanted to do that, but it interferes with keeping this flag optional. If this option is set, it takes precedent over Engine API version loading via `debug_chainConfig`. Is it possible to set a default and then activate this flag by using it without a value?

Reply:
You could check `IsSet`. I believe that won't trip if a default value has been used.

Reply:
I just realized that I already use `IsSet` and you're right, it is `false` if the flag is not provided, even if the `Value` field is set. However, I can never access that default value, because it's not possible to use the flag `--engine.version` without a value. It gives the error `flag needs an argument: -engine.version`. So decided against setting a default, as it's inaccessible.

Reply:
The default is used if the flag is not set. This flag is not required. So for a user they would just skip the `-engine.version` flag all toghether.

Reply:
If the flag is unset, its value is also not used, because then the engine version is read from the chain config via `debug_chainConfig`. That's why I meant that the default value wouldn't have any effect. See setup code in [`initVersionProvider`](https://github.com/ethereum-optimism/optimism/pull/9681/files#diff-817c7fb9158c0f338af419b307f69da3ccf5d796275310842b052547e075ee1cR190).